provider/doc_edits.go

```diff
@@ -7,7 +7,9 @@ import (
 )
 
 func editRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
-	return append(defaults, fixupEffectiveLabels)
+	var result = append(defaults, fixupEffectiveLabels)
+	result = append(result, removeSecretsInPlainTextNote)
+	return result
```

Comment:
`append` takes `(src []T, elems ...T)`.
```suggestion
	return append(defaults, 
		fixupEffectiveLabels,
		removeSecretsInPlainTextNote,
	)
```